src/security-council-mgmt/governors/SecurityCouncilNomineeElectionGovernor.sol

```diff
@@ -223,8 +223,8 @@ contract SecurityCouncilNomineeElectionGovernor is
         }
 
         ProposalState state_ = state(proposalId);
-        if (state_ != ProposalState.Active) {
-            revert ProposalNotActive(state_);
+        if (state_ != ProposalState.Pending) {
```

Comment:
for how long is the proposal pending before it becomes active? 

I know they have deterministic start times, but im not sure how deterministic the pending state is (especially considering edge case where previous elections might have been delayed in executing their results).

Reply:
If we're concerned about this, maybe we could make it cover both the pending and active period

```suggestion
        if (state_ != ProposalState.Pending) {
```

Reply:
would be good to add documentation that makes the above clear

Reply:
iirc the plan was to set the pending duration (`votingDelay`) to a week. that would give us a week for addContender and a week for voting for nominees. in the current impl you wouldn't be able to add new contenders during that second week but it would be easy to change

Reply:
> but im not sure how deterministic the pending state is (especially considering edge case where previous elections might have been delayed in executing their results).

the previous election must be executed from the member governor before the current one starts. the pending phase starts at whatever block `createElection` is called.

Reply:
Should we add a note in this part of the code that says where the pending duration is set?
Will that value be updated as part of the Action contract?

Reply:
yes it's updated [here](https://github.com/ArbitrumFoundation/governance/pull/231/files#diff-18e2540570c8d06729616883e29e10ce4bb0272e0ccf79fd850f58e9e9e0fe20R47)